participants/adieg.json

```diff
@@ -0,0 +1,5 @@
+{
+  "Name": "Adi Egozi",
+  "linkedin": "Adi Egozi",
```

Comment:
This is not your username on linkedin. this `adi-egozi-802674227` is. (and you could get rid of those numbers as well

Reply:
See https://szabgab.com/beautify-your-linkedin-profile-link.html

Reply:
Thanks! I was trying to find some kind of unique identifier in my linkedin and couldn't find it.

Reply:
Oh so it was unclear tha the URL is the unique id